src/pages/homePage.js

```diff
@@ -0,0 +1,26 @@
+// /* eslint-disable max-len */
```

Comment:
- [ ] Kindly delete the unused line of code to have a clean and professional codebase 🙃 😉